core/src/model/IEnvironmentCache.java

```diff
@@ -0,0 +1,43 @@
+package model;
+
+import com.badlogic.gdx.maps.Map;
+import com.dongbat.jbump.World;
+
+/**
+ * An interface that should allow for different implementations of maps, not just TiledMaps
+ * but also potentially things like spreadsheets or just image files. For 2D games, tiles are
+ * at this time ubiquitous and considered the standard but this might change in the future
+ */
+public interface IEnvironmentCache {
+
+    /**
+     * Gets the JBump "world" which is used for moving entities and registering collisions
+     * @return the JBump world
+     */
+    World<IEntity> getWorld();
+
+    /**
+     * Used to make sure the map used by the view is the same as the model
```

Comment:
I'm not sure if the view and the model should be mentioned in the documentation for this method. The documentation should say what the method does.